python/lib/sift_grafana/sift_query_model.py

```diff
@@ -0,0 +1,56 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, Field, model_validator
+from typing_extensions import Self
+
+
+# These are the models that are used to validate the input data for the Sift Grafana Plugin queries
+class Channel(BaseModel):
+    channelId: Optional[str] = None # noqa: N815
```

Comment:
Could you change this to:

```python
class Channel(BaseModel):
    channel_id: Optional[str]
    channel_identifier: Optional[str]
    
    def __init__(self, channel_id: Optional[str] = None, channel_identifier: Optional[str] = None):
        ...
```

Reply:
Unfortunately the main use case with pydantic validation doesn't seem to play nicely with the init style.

`sift_query = SiftQuery.model_validate(from_json(sift_query_json, allow_partial=True))` 

Definitely open to other ways to maintain the standard though